docs/sphinx/source/whatsnew/v0.7.0.rst

```diff
@@ -10,6 +10,14 @@ compatibility notes.
 **Python 2.7 support ended on June 1, 2019**. (:issue:`501`)
 **Minimum numpy version is now 1.10.4. Minimum pandas version is now 0.18.1.**
 
+API Changes
+~~~~~~~~~~~
+* Moved functions for cell temperature from `pvsystem.py` to `celltemp.py`, renamed from `pvsystem.sapm_celltemp` to `celltemp.sapm`, and from `pvsystem.pvsyst_celltemp` to `celltemp.pvsyst`. Argument order in `celltemp.sapm` changed and output type from `celltemp.pvsyst` have been changed to be consistent in `celltemp.py` for all cell temperature model functions. (:issue:`678')
```

Comment:
2nd sentence is hard for me to parse. Perhaps two sentences.

Reply:
Need to mention the `PVSystem.sapm_celltemp` method API change.

Reply:
The changing argument order is tough. We can, in principle, make the deprecation decorator swap the arguments for the old function when passing them to the new function. But I don't know how to do that for the method wrapper. I hate to break user code without a warning cycle, but we might not have a choice.

Reply:
We could rename the method wrapper.